apps/admin-gui/src/app/shared/components/roles-list/roles-page.component.html

```diff
@@ -27,6 +26,7 @@ <h1 class="page-subtitle d-flex">{{'ROLES.TITLE'| translate}}</h1>
         matTooltipPosition="left"
         [matTooltipDisabled]="(selection.selected.length === 0 && selectedFacilities.selected.length === 0) || !disableRemove">
         <button
+          *ngIf="editable"
```

Comment:
Nitpick but the proper target should be the parent of the button (span tag). Right now the span is present so the tooltip could be triggered.